documentation/DATA_FORMAT.md

```diff
@@ -158,11 +158,16 @@ Besides the details shown above, these are additional parameters that the manife
 {
     ...
     "metadata": {
-        "frameDims": {
-            "units": <unit label for frame dimensions>,
-            "width": <width of frame in units>,
-            "height": <height of frame in units>
-        },
```

Comment:
Note breaking change here, because the `dataclasses-json` library I started using doesn't have any easy way to do mapping. I have a design question about this, so let me spell out what this means:

BEFORE:
`ColorizerMetadata` accepted `frame_units`, `frame_width`, and `frame_height` arguments and wrapped them together into `frameDims` in the final output:

```
# Client API
metadata = ColorizerMetadata(frame_units = "a", frame_width = 10.0, frame_height = 20.0)

# Data API
metadata.to_json()
"""
{
    "frameDims": {
        "units": "a",
        "width": 10.0,
        "height": 20.0,
    }
}
"""
```

I can either change the client format or the data format (what I'm currently doing).

**Changing client format:**
```
metadata = ColorizerMetadata(frame_dims={"width": 10.0, "height": 20.0, "units": "a"})
```
**Changing data format:**
```
"""
{
    "frameUnits": "a",
    "frameWidth": 10.0,
    "frameHeight": 20.0,
}
"""
```

I **think** the second pattern is cleaner and I won't have to change any of the data scripts, but the tradeoff is that the colorizer UI will need to handle both the new and old data format. Any thoughts?

Reply:
it's not clear to me why you need to change it.  Is it because `dataclasses-json` doesn't play nice with it?

Reply:
Got a working solution! (it's a little bit hacky, but there's unit tests for it.) Closing this :')

Reply:
My only preference is to not let the third party tool dictate how we want our data format to look.  And I think your working solution accomplishes that :)